REQUIRE

```diff
@@ -1,4 +1,5 @@
-julia 0.4
+julia 0.4+
```

Comment:
I think in the first iteration I was planning on removing support vor `v0.4` completely before settling on `Compat`.